### PR TITLE
<HELIOS> Add Debug and error prints

### PR DESCRIPTION
**Pull request by pranjalchanda08**

### CODE REVIEW

File: src/visor/terravisor/services/kernel/helios_sched.c [hunk 1]
```diff
@@ -61,19 +65,24 @@ helios_sched_ctrl_t g_sched_ctrl =
  *	INTERNAL USED FUNCTIONS (NON-STATIC)
  *****************************************************/
 
+void _helios_sched_panic(const char* reason)
+{
+	syslog(fail, "HELIOS Panic: %s", reason);
+	arch_panic_handler();
```
Comment (akashkollipara) [inline src/visor/terravisor/services/kernel/helios_sched.c:71]: `plat_panic_handler` would be more reasonable one...

File: src/visor/terravisor/services/kernel/helios_sched.c [hunk 2]
```diff
@@ -360,7 +366,7 @@ void _helios_scheduler_despatch(void)
 	else
 	{
 		/* A System Error case */
-		arch_panic_handler();
+		_helios_sched_panic("Presched Callback not provided!");
```
Comment (akashkollipara) [inline src/visor/terravisor/services/kernel/helios_sched.c:369]: may be use status_t values?

**Review by akashkollipara — CHANGES_REQUESTED** [review]

File: src/visor/terravisor/services/kernel/helios_sched.c (hunk 1 above)
Comment (pranjalchanda08) [reply src/visor/terravisor/services/kernel/helios_sched.c:71]: Done

File: src/visor/terravisor/services/kernel/helios_sched.c (hunk 2 above)
Comment (pranjalchanda08) [reply src/visor/terravisor/services/kernel/helios_sched.c:369]: We do not want to return a error status here to the user. It shall directly panic.

Comment (akashkollipara) [reply src/visor/terravisor/services/kernel/helios_sched.c:369]: no i was thinking something like https://github.com/VisorFolks/cyancore/blob/stable/src/driver/driver.c#L205

Comment (pranjalchanda08) [reply src/visor/terravisor/services/kernel/helios_sched.c]: Updated

**Pull request merged**